components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java

```diff
@@ -214,11 +216,41 @@ public static IdentityCookieConfig getIdentityCookieConfig(String cookieName) {
         return identityCookiesConfigurationHolder;
     }
 
+    /**
+     * This method can use to check whether the legacy feature for the given legacy feature id is enabled or not
+     *
+     * @param legacyFeatureId      Legacy feature id.
+     * @param legacyFeatureVersion Legacy feature version.
+     * @return Whether the legacy feature is enabled or not.
+     */
+    public static boolean isLegacyFeatureEnabled(String legacyFeatureId, String legacyFeatureVersion) {
+
+        String legacyFeatureConfig = legacyFeatureId.trim() + legacyFeatureVersion.trim();
```

Comment:
add nullcheck on legacyFeatureId and legacyFeatureVersion. If Id is null, return. However feature version can be allowed to be null in the case of identity/connect/dcr. That should be handled properly

Reply:
Fixed 76ad8e0

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3027/commits/76ad8e0307b509aefc1c7e7716756ad143850105